tsconfig.packages.json

```diff
@@ -15,6 +15,7 @@
     "paths": {
       "@ts-bridge/*": ["../*/src/index.ts"]
     },
+    "skipLibCheck": true,
```

Comment:
The latest `@types/node` isn't compatible with TypeScript pre-`5.8`(?). Seems like the easiest solution for now is to disable `skipLibCheck`, since we don't use the features that cause the TypeScript errors (mainly typed arrays).